packages/safe-apps-react-sdk/tsconfig.json

```diff
@@ -1,22 +1,7 @@
 {
```

Comment:
this package was incorrectly using its own configuration instead of the workspace one